File: Pratica 6 - Funcoes 2/cpp_logic/exercicio1_v1.cpp
#include <iostream>
using namespace std;

int main() {
    int num;
    cout << "Digite um número inteiro: ";
    cin >> num;

    if (num % 2 == 0)
        cout << "Par";
    else
        cout << "Ímpar";

    return 0;
}

File: Pratica 6 - Funcoes 2/java_logic/Exercicio1_v1.java
import java.util.Scanner;

public class Exercicio1_v1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Digite um número inteiro: ");

        int num = scanner.nextInt();

        if (num % 2 == 0) {
            System.out.println("Par");
        } else {
            System.out.println("Ímpar");
        }

        scanner .close();
    }
}
